server/athenian/api/models/state/models.py

```diff
@@ -421,7 +421,7 @@ class TeamGoal(create_time_mixin(created_at=True, updated_at=True), Base):
         nullable=False,
         primary_key=True,
     )
-    target = Column(JSON, nullable=False)
+    target = Column(JSONB().with_variant(JSON(), sqlite.dialect.name), nullable=False)
```

Comment:
Let's define this type as JSON in the module, we will never need the original JSON type:
```
from sqlalchemy import JSON, JSONB
JSON = JSONB().with_variant(JSON(), sqlite.dialect.name)  # erase the original type
```

Reply:
used an alias to keep mypy happy
There were two other simple JSON columns now converted